### PR TITLE
Initial cypress examples for cheatsheet

### PR DESCRIPTION
**Pull request by Devonjedi**
## Description
Have updated administration and creating a node section of cheatsheet.md. 

Also need to test the examples code out for creating a node portion and validate that these cypress snippets are appropriate examples.

Looking to create another repo or sandbox site to test out these cypress commands/ examples for creating a node as a way to see it in action or to experiment with cypress

## Related Tickets
https://kanopi.teamwork.com/app/tasks/29311887

## Steps to Validate
1. None yet

### CODE REVIEW

File: CHEATSHEET.md [hunk 1]
```diff
@@ -3,25 +3,118 @@
...
+Cypress.Commands.add("autocomplete", (selector, content) => {
+  cy.get(selector).type(content).then(($input) => {
+    cy.get('.ui-autocomplete:visible a:first').click();
+  });
+});
+
+
 * Fill out a rich text editor field
+** Use ckeditorType command 
+Cypress.Commands.add("ckeditorType", (selector, content) => {
```
Comment (paulsheldrake) [inline CHEATSHEET.md]: Link to the support command src code instead of copying it

File: CHEATSHEET.md [hunk 2]
```diff
@@ -3,25 +3,160 @@
 ## Administration
 
 * Log into Drupal
```
Comment (thejimbirch) [inline CHEATSHEET.md]: Can you change all these bullets to H3s?

### Log into Drupal

File: CHEATSHEET.md [hunk 3]
```diff
@@ -3,25 +3,160 @@
...
+[Link to login.js](login.js)
+cy.login('User', 'pass')
```
Comment (thejimbirch) [inline CHEATSHEET.md]: For single line code, wrap them in single `

`cy.login('User', 'pass')`

File: CHEATSHEET.md [hunk 4]
```diff
@@ -3,25 +3,160 @@
...
+** Use Cypress autocomplete command will select first option that appears for autocomplete
+[Link to autocomplete](autocomplete.js)
+cy.autocomplete('[data-drupal-selector="edit-field-sample-0-target-id"]', 'item');
+
 * Fill out a rich text editor field
+** Use ckeditorType command 
+[Link to ckeditortype](ckeditorType.js)
+cy.ckeditorType('#edit-body-0-value', 'string of text')
+
 * Create a Paragraph
```
Comment (thejimbirch) [inline CHEATSHEET.md]: For multiple line code blocks, ass ```

```
cy.visit('/node/123/edit');

cy.get('.add-paragraph-button').click();

cy.get('#edit-paragraph-field-name').type('My Paragraph Name');
// Fill in other paragraph fields here
cy.get('#edit-submit').click();
```

**Review by thejimbirch — CHANGES_REQUESTED** [review]

**Review by thejimbirch — APPROVED** [review]
Great job, thanks!

**Pull request merged**